### DISCUSSION TITLE
Do maint releases get published?

### DISCUSSION THREAD

**Discussion by hcldan** [bergie/passport-saml#871] [General]
There's a fix for a security vulnerability that got made on the 3x stream in April but no signs of a release for it.

**Comment by cjbarth** [comment]
Please see this recommendation: https://github.com/node-saml/passport-saml/pull/857#issuecomment-1517876269

**Reply by hcldan** [reply]
yeah, that's what I ended up doing.  Was hoping for a release but I'll try to upgrade to 4x when I get a chance to look at what changed.
